_po/ja/how-to/laravel.po

```diff
@@ -597,6 +904,14 @@ msgid ""
 "                                            </dd>\n"
 "                                        </div>"
 msgstr ""
+"                                        <div class=\"border-t border-gray-200 p"
+"t-1 sm:col-span-2\">\n"
+"                                            <dd class=\"sm:mt-2 text-sm text-gr"
+"ay-500\">\n"
+"                                                <span class=\"block m-1\">{{ $po"
+"st->title }}</span>\n"
+"                                            </dd>\n"
+"                                        </div>"
```

Comment:
Probably, this is needless.
Because this test is same as before translation text.